packages/anon-aadhaar-contracts/test/Verifier.ts

```diff
@@ -56,130 +51,29 @@ describe('VerifyProof', function () {
-        ).toString()
-
-        const wrongAnonAadhaarVerifier = AnonAadhaarVerifier.deploy(
-          _verifierAddress,
-          wrongAppId,
-        )
-
-        await expect(wrongAnonAadhaarVerifier).to.be.revertedWith(
-          'AnonAadhaarVerifier: group id must be < SNARK_SCALAR_FIELD',
-        )
-      })
-
-      it('Should revert for a valid proof with the wrong public key', async function () {
-        const { anonAadhaarVerifier, appIdBigInt } = await loadFixture(
-          deployOneYearLockFixture,
-        )
-
-        const testData: [bigint, bigint, bigint, bigint] = await genData(
-          'Hello world',
-          'SHA-1',
-        )
-
-        const input = {
-          signature: splitToWords(BigInt(testData[1]), BigInt(64), BigInt(32)),
-          modulus: splitToWords(BigInt(testData[2]), BigInt(64), BigInt(32)),
-          base_message: splitToWords(
-            BigInt(testData[3]),
-            BigInt(64),
-            BigInt(32),
-          ),
-          app_id: appIdBigInt,
-        }
-
-        const dirName = __dirname + '/../../anon-aadhaar-pcd/artifacts/RSA'
-
-        const { a, b, c, Input } = await exportCallDataGroth16(
-          input,
-          dirName + '/main.wasm',
-          dirName + '/circuit_final.zkey',
-        )
-
-        expect(
-          anonAadhaarVerifier.verifyProof(a, b, c, Input),
-        ).to.be.revertedWith('AnonAadhaarVerifier: wrong issuer public key')
-      })
 
-      it('Should revert for a valid proof with wrong app_id', async function () {
-        const { anonAadhaarVerifier } = await loadFixture(
-          deployOneYearLockFixture,
-        )
-
-        const testData: [bigint, bigint, bigint, bigint] = await genData(
-          'Hello world',
-          'SHA-1',
-        )
-
-        const app_id = BigInt(
-          parseInt(crypto.randomBytes(20).toString('hex'), 16),
-        ) // random value.
-
-        const input = {
-          signature: splitToWords(BigInt(testData[1]), BigInt(64), BigInt(32)),
-          modulus: splitToWords(BigInt(testData[2]), BigInt(64), BigInt(32)),
-          base_message: splitToWords(
-            BigInt(testData[3]),
-            BigInt(64),
-            BigInt(32),
-          ),
-          app_id: app_id.toString(),
-        }
-
-        const dirName = __dirname + '/../../anon-aadhaar-pcd/artifacts/RSA'
-
-        const { a, b, c, Input } = await exportCallDataGroth16(
-          input,
-          dirName + '/main.wasm',
-          dirName + '/circuit_final.zkey',
-        )
-
-        await expect(
-          anonAadhaarVerifier.verifyProof(a, b, c, Input),
-        ).to.be.revertedWith('AnonAadhaarVerifier: wrong app ID')
-      })
-
-      //   it('Should return true for a valid proof', async function () {
-      //     const { anonAadhaarVerifier, appIdBigInt } = await loadFixture(
+      //   it('Should return true for a valid proof with webProver', async function () {
```

Comment:
You can use `it.skip()` here, we ensure we always remember this test.

Reply:
I will do that